script.js

```diff
@@ -1,4 +1,216 @@
+// Black Jack rules:
+// The player with 2 ace win
+// The player with highest score but <= 21 wins
+// Ace consider as 11 when there is only 2 cards
+// Ace consider as 1 if there are more than 2 cards
+
+
+// The game mode
+
+var start_game = 'start game';
+var hit_or_stand = 'hit or stand';
+var gameMode = start_game;
+
+// The players
+var player = [];
+var comp = [];
+
+//Define var for player sum
+var totalPlayerSum = Number();
+var totalCompSum = Number();
```

Comment:
Number() needs and object for example: Number(object)
Read more about Number method here: https://www.w3schools.com/jsref/jsref_number.asp